thirdparty/hairer/radau5_c_py.pyx

```diff
@@ -83,7 +83,10 @@ cdef int callback_jac(integer* n, doublereal* x, doublereal* y, doublereal* fjac
     cdef np.ndarray[double, ndim=1, mode="c"]y_py_in = np.empty(n[0], dtype = np.double)
     c2py(y_py_in, y, n[0])
     res = (<object>jac_PY)(x[0], y_py_in)
-    py2c_matrix_flat_F(fjac, res, res.shape[0], res.shape[1])
+    ipar[0] = res[1][0]
+    if ipar[0] != 0:
+        return 0
+    py2c_matrix_flat_F(fjac, res[0], res[0].shape[0], res[0].shape[1])

```

Comment:
What is this a fix for or is it a update to a new version of a third party?

Reply:
This is an update and would need to be looked at in combination with the changes in radau_decsol_c.c

ipar is an integer return value for in the callback, which was previously unused in the c code. 
I moved this before the py2c_matrix_flat_F class, since a nonzero value in ipar[0] (from C) will reject the timestep, which makes the other outputs of this callback irrelevant.

Reply:
okay